src/main.py

```diff
@@ -636,18 +623,20 @@ def replica_return(self, db_state, zk_state):
         my_hostname = helpers.get_hostname()
         self.write_host_stat(my_hostname, db_state)
         holder = zk_state['lock_holder']
+        logging.debug('Replica is returning. So we resume WAL replay to {}'.format(holder))
```

Comment:
This is superfluous, we have almost same message at [R632](https://github.com/yandex/pgconsul/pull/59/files#diff-2e5ad92c43aa96cc3a9cef6c6aec998b216f1379c43b1f651013d25e55989312R632), and it is better there as intent logging.